db/seeds.rb

```diff
@@ -1,7 +1,50 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+require 'csv'
+require 'faker'
+
+# add cities in DB
+
+City.all.destroy_all
+
+source_file = "/Users/eleonore/Documents/Code/enm-city-allocator/db/enm_cities_seats.csv"
+
+CSV.foreach(source_file, headers: true) do |row|
+	City.create(
+		name: row[1],
+		nb_seats: row[2]
+		)
+end
+
+# add students in DB
+
+Student.all.destroy_all
+
+Student.create(name: "Anna Thuleau")
+
+335.times {
+	Student.create!(
+	name: Faker::Name.unique.name
+	)
+}
+
+# add choices in DB
+
+Choice.all.destroy_all
+
+Student.all.each do |student|
+	rank = 1
+	6.times {
+		n = rand(100)
+		if n < 20
+			random_city = City.find_by name: 'PARIS (PI)'
+		elsif n < 40
+			random_city = City.find_by name: 'BORDEAUX (PI)'
+		else
+			random_city = City.all.sample(100)[50]
```

Comment:
```suggestion
			random_city = City.all.sample
```
will already give you a random city